files/machine_provision.yml

```diff
@@ -98,6 +99,13 @@
                           initrd_desc="CentOS 7.5"
                           preseed_name="CentOS.Upstream"
                           kernel_opts="ip=dhcp text inst.stage2=http://10.40.0.13/mirror.centos.org/altarch/7/os/aarch64/ inst.repo=http://10.40.0.13/mirror.centos.org/altarch/7/os/aarch64/ inst.ks=file:/ks.cfg"
+                        elif [ "${os_type}" == "opensuse" ]; then
+                          preseed_type="autoyast"
+                          kernel_desc="openSUSE Leap 15.1"
+                          initrd_desc="openSUSE Leap 15.1"
+                          preseed_name="openSUSE Leap 15.1"
```

Comment:
Change the preseed name to "OpenSUSE.Upstream" to match @BaptisteGerondeau style and so that we can change the underlying files without needing to re install Jenkins Jobs.

Reply:
Done.